resources/lang/ru.json

```
@@ -363,11 +364,32 @@
     "Choose date": "Выбрать дату",
     "The :resource was updated!": "Ресурс :resource обновлён!",
     "The resource was updated!": "Ресурс обновлён!",
+    "The :resource was deleted!": "Ресурс :resource удален!",
+    "The :resource was restored!": "Ресурс :resource восстановлен!",
     "Increase": "Увеличить",
     "Constant": "Оставить как есть",
     "Decrease": "Уменьшить",
     "Reset Password Notification": "Оповещение о сбросе пароля",
     "Nova User": "Пользователь Nova",
+    "of": "of",
```

You left this untranslated. It is used like this: `{{ page }} of {{ totalPages }}`.

Is this suggestion below correct?
```suggestion
    "of": "из",
```